product_docs/docs/pgd/5/rel_notes/pgd_5.2.0_rel_notes.mdx

```diff
@@ -0,0 +1,72 @@
+This version addresses security vulnerabilities in dependencies for PGD Proxy and PGD CLI.
+
+## Highlights of EDB  Postgres Distributed 5.2
+
+*
+
+!!! Important Recommended upgrade
+    We recommend that users of PGD 5.1 upgrade to PGD 5.2.
+
+!!! Note PostgreSQL version compatibility
+This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8, and later.
+!!!
+
+
+| Component | Version | Type            | Description  |
+|-----------|---------|-----------------|--------------|
+
+| BDR       | 5.2.0   | Feature         | Sub-group Raft |
+| BDR       | 5.2.0   | Feature         | Improve handling of concurrent workloads with parallel apply |
+| BDR       | 5.2.0   | Feature         | Parallel apply for synchronous commit scopes (Eager, CAMO, Group Commit)
+| BDR       | 5.2.0   | Enhancement     | Enhance Parallel Apply for deadlock mitigation |
+| BDR       | 5.2.0   | Enhancement     | Introduce receiver side transaction state tracking |
+| BDR       | 5.2.0   | Enhancement     | Implement sync_node capability |
+| BDR       | 5.2.0   | Enhancement     | Release tuple set for CAMO pre-commit |
+| BDR       | 5.2.0   | Enhancement     | Allow multiple SYNCHRONOUS_COMMIT clauses in a Commit Scope rule |
+| BDR       | 5.2.0   | Bug             | Change to group_raft_details view to avoid deadlock possibility |
+| BDR       | 5.2.0   | Bug             | Release tuple set for CAMO pre-commit |
+| BDR       | 5.2.0   | Bug             | Remove bdr.sync_node() and supporting functions from PGD 5.2 |
+| BDR       | 5.2.0   | Bug             | More clear log during extension upgrade|
+| BDR       | 5.2.0   | Bug             | Add check for conflicting node names |
+| BDR       | 5.2.0   | Bug             | Fix a crash during Raft snapshot restore |
+| BDR       | 5.2.0   | Bug             | Ensure ACTIVE nodes do no longer attempt connecting to PARTING ones |
+| BDR       | 5.2.0   | Bug             | Fix `tcp_user_timeout` GUC to use the correct unit |
+| BDR       | 5.2.0   | Bug             | Correct v26 snapshots written by BDR 4 to be readable by 3.7 |
+| BDR       | 5.2.0   | Bug             | Modify bdr.subscription_stat for new columns |
+| BDR       | 5.2.0   | Bug             | miscellaneous fixes for PostgreSQL 16 compatibility |
+| BDR       | 5.2.0   | Bug             | Address a crash when BDR is used together with pgaudit |
+| BDR       | 5.2.0   | Bug             | Use the new async 2PC for group commit |
+| BDR       | 5.2.0   | Bug             | Skip parting synchronization to witness node |
+| BDR       | 5.2.0   | Bug             | Fix a bug in connection string construction |
+| BDR       | 5.2.0   | Bug             | Enable receiver side transaction tracking for non-streaming CAMO transactions |
+| BDR       | 5.2.0   | Bug             | Allow commit/abort prepared to create a transaction state |
+| BDR       | 5.2.0   | Bug             | Release tuple set for CAMO pre-commit |
```

Comment:
Please remove; this was bug fix for 5.2 development of PA enhancement. So customer would not have seen it.